catalogue_pipeline/ingestor/src/main/scala/uk/ac/wellcome/platform/ingestor/services/WorkIndexer.scala

```diff
@@ -49,19 +50,26 @@ class WorkIndexer(
       .execute {
         bulk(inserts)
       }
-      .map { bulkResponse: BulkResponse =>
-        val actualFailures = bulkResponse.failures.filterNot {
-          isVersionConflictException
-        }
+      .map { response: Response[BulkResponse] =>
+        if (response.isError) {
+          warn(s"Error from Elasticsearch: $response")
+          Right(works)
```

Comment:
Why right here? doesn't that cause messages to be deleted from the queue even though there was an elasticsearch failure?

Reply:
Because I am a goofball and meant `Left`, I think. Oops. Will fix when I get in.